ConfigurationSystem/Client/Utilities.py

```diff
@@ -486,11 +486,9 @@ def getDBParameters( fullname, defaultHost = 'localhost',
   'db' and 'queueSize'
   """
 
-  fullname = fullname
```

Comment:
You need to assign the default values like it was done. So if there is no specific value, and no general value in the CS, then we take the one given in parameters
With your fixes, the values given in parameters are never used